app/search_builders/curation_concerns/collection_search_builder.rb

```diff
@@ -1,33 +1,48 @@
 module CurationConcerns
+  # Our parent class is the generated SearchBuilder descending from Blacklight::SearchBuilder.
+  # Defines search_params_logic used when searching for Collections.
+  # @see https://github.com/projecthydra/curation_concerns/blob/master/app/search_builders/curation_concerns/README.md SearchBuilders README
+  #
   class CollectionSearchBuilder < ::SearchBuilder
-    # Defines which search_params_logic should be used when searching for Collections
-    self.default_processor_chain = [:default_solr_parameters, :add_query_to_solr,
```

Comment:
Removing these is intentional, since we are relying on the common chain built up in `SearchBuilder`.   Instead of needing to re-specify a new chain w/ bespoke methods, we are using (overriding) more of the already defined methods (e.g. `filter_models` instead of `add_collection_filters`).